canister/src/main.rs

```diff
@@ -44,6 +44,14 @@ pub fn bitcoin_get_balance(request: GetBalanceRequest) {
     }
 }
 
+#[query(composite = true, manual_reply = true)]
```

Comment:
Based on the discussions here, we've updated the interface spec to use regular queries and not composite queries. To enforce that queries are only called in non-replicated mode, we can implement [this trick](https://dfinity.slack.com/archives/C01D7R95YJE/p1691418227396519?thread_ts=1691417215.202529&cid=C01D7R95YJE).

Reply:
It has changed now. PTAL